src/sentry/replaystore/bigtable/backend.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+import os
+from datetime import timedelta
+from typing import Any, Dict, List, Tuple
+
+from sentry.replaystore.base import ReplayNotFound, ReplayStore
+from sentry.utils.kvstore.bigtable import BigtableKVStorage
+from sentry.utils.strings import compress
+
+
+class BigTableReplayStore(ReplayStore):
+    store_class = BigtableKVStorage
+
+    def __init__(
+        self,
+        project: str | None = None,
+        instance: str = "sentry",
+        table: str = "replaystore",
+        automatic_expiry: bool = False,
+        default_ttl: timedelta | None = None,
+        compression: str | bool | None = None,
+        **client_options: Dict[Any, Any],
```

Comment:
I assume that these are at least `Dict[str, Any]`. But we could probably type the values better as well, since I assume it'd be something along the lines of str, int, etc?

Reply:
the type in BigTableKVStore is `Mapping[Any,Any]`, i've switched this type to be the same. it looks like the GCP bigtable has a ClientOptions type that we could use, but i think outside of the scope for now. i can add a TODO. https://googleapis.dev/python/google-api-core/latest/client_options.html#google.api_core.client_options.ClientOptions